README.md

```diff
@@ -117,33 +117,40 @@ The following prerequisites are required:
 With your terminal go into the `coatrack/docker` directory where you will find the script `build-and-deploy-images-locally.sh` which builds CoatRack from source, creates container images on your local machine and runs CoatRack locally using these images. This is especially useful for developers who did changes to the source code and want to locally test the impact of these changes in a realistic setup.
 
 ## Use FIWARE Keyrock for Identity Management instead of Github
-By default, CoatRack uses Github for identity management and login. As an alternative, you can set up CoatRack to use KeyRock IdM, which is the FIWARE enabler for Identity Management. In this case you have to set up KeyRock first and then set up CoatRack.
-Moreover, the example shown below will consider that you deployed CoatRack in `localhost:8080`, and that Keyrock was deployed via Docker with the version 7.8.1 (fiware/idm:7.8.1).
+By default, CoatRack uses Github for identity management and login. As an alternative, you can set up CoatRack to use [Keyrock](https://keyrock-fiware.github.io/), which is the FIWARE enabler for Identity Management. In this case you have to set up Keyrock first and afterwards configure CoatRack to work with your Keyrock instance.
+The example set-up explained below is based on the following assumptions: 
+
+* Keyrock version 7.8.1 is deployed via Docker (image `fiware/idm:7.8.1`)
+* Keyrock is accessible via `localhost:3000`
+* CoatRack is accessible via `localhost:8080`
 
 Based on this, in order to make CoatRack work with the FIWARE Keyrock, you will need to:
 
-- First, pull and deploy the containers of keyrock via docker (as the Keyrock itself needs to be running in your local machine). To do this, just follow
-the tutorial in https://fiware-idm.readthedocs.io/en/latest/getting_started/index.html
+- At first you have to set-up/deploy your Keyrock instance. To do this, just follow the tutorial in https://fiware-idm.readthedocs.io/en/latest/getting_started/index.html
 - Please check that Keyrock is running correctly in your local machine by checking if you can access the Keyrock GUI,
 - If the Keyrock container is running and you can access the Keyrock GUI, then we need to start by registering CoatRack as application in Keyrock, to do this:
   - Create a new user account or log in with the account `admin@test.com`, which is created as the administrator account by default,
```

Comment:
To be clarified: Can a normal user account register an application, or only admin? In any case, please remove the concrete user name `admin@test.com`, as this could change in the future.

Reply:
A normal user can register an application

Reply:
Done, so I removed the `admin@test.com`, and just rephrased it to "admin account"